src/pybindings/pycontroller/ControllerBindings/PyChip_BLEManager.cpp

```diff
@@ -0,0 +1,148 @@
+#include <platform/CHIPDeviceLayer.h>
+#include <platform/PlatformManager.h>
+#include <sstream> // __str__
+#include <system/SystemLayer.h>
+
+#include <functional>
+#include <pybind11/pybind11.h>
+#include <string>
+
+#ifndef BINDER_PYBIND11_TYPE_CASTER
+#define BINDER_PYBIND11_TYPE_CASTER
+PYBIND11_DECLARE_HOLDER_TYPE(T, std::shared_ptr<T>)
+PYBIND11_DECLARE_HOLDER_TYPE(T, T *)
+PYBIND11_MAKE_OPAQUE(std::shared_ptr<void>)
+#endif
+
+void bind_PyChip_BLEManager(std::function<pybind11::module &(std::string const & namespace_)> & M)
```

Comment:
is this generated files? if yes, please put all generated file under gen folder, and also add readme file to show how to run codegen.

Reply:
I will add a REAME on how to use binder. I will also include how to update the name, because binder just sets numbers.

Reply:
https://github.com/krypton36/connectedhomeip/blob/PyBind11ControllerWrapper/src/pybindings/Bnder_Code_Gen.md